client/src/app/licitacoes/licitacoes-detalhar/licitacoes-detalhar.component.html

```diff
@@ -9,31 +9,81 @@ <h3>{{ licitacao?.nm_orgao }}</h3>
     <strong>Objeto da licitação: </strong> <span class="text-muted">{{ licitacao?.descricao_objeto }}</span>
   </p>
   <br>
-  <table class="table table-striped">
-    <thead>
-    <tr>
-      <th scope="col">#</th>
-      <th scope="col">Item</th>
-      <th scope="col">Quantidade de itens</th>
-      <th scope="col">Unidade</th>
-      <th scope="col">Valor unitário estimado</th>
-      <th scope="col">Valor toal estimado</th>
-    </tr>
-    </thead>
-    <tbody>
-    <tr *ngFor="let item of itens; index as i">
-      <th scope="row">{{ i + 1 }}</th>
-      <td>
-        {{ item.ds_item }}
-      </td>
-      <td>{{ item.qt_itens_licitacao  }}</td>
-      <td>{{ item.sg_unidade_medida }}</td>
-      <td>{{ item.vl_unitario_estimado | currency:'R$ ' }}</td>
-      <td>{{ item.vl_total_estimado | currency:'R$ ' }}</td>
-    </tr>
-    </tbody>
-  </table>
+  <div>
+    <h5>Itens da Licitação</h5>
+    <table class="table table-striped">
+      <thead>
+        <tr>
+          <th scope="col">#</th>
+          <th scope="col">Item</th>
+          <th scope="col">Quantidade de itens</th>
+          <th scope="col">Unidade</th>
+          <th scope="col">Valor unitário estimado</th>
+          <th scope="col">Valor toal estimado</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr *ngFor="let item of itensPaginate; index as i">
+          <th scope="row">{{ i + 1 }}</th>
+          <td>
+            {{ item.ds_item }}
+          </td>
+          <td>{{ item.qt_itens_licitacao  }}</td>
+          <td>{{ item.sg_unidade_medida }}</td>
+          <td>{{ item.vl_unitario_estimado | currency:'R$ ' }}</td>
+          <td>{{ item.vl_total_estimado | currency:'R$ ' }}</td>
+        </tr>
+      </tbody>
+    </table>
+    <div class="d-flex justify-content-center">
+      <ngb-pagination [collectionSize]="licitacao.itensLicitacao.length" [(page)]="page" [pageSize]="pageSize">
+      </ngb-pagination>
+    </div>
+  </div>
+  <br>
+  <div>
+    <h5>Contratos</h5>
+    <div class="card" *ngFor="let contrato of licitacao.contratosLicitacao ; index as i;">
```

Comment:
```suggestion
      <ngb-pagination [collectionSize]="licitacao?.itensLicitacao?.length" [(page)]="page" [pageSize]="pageSize">
      </ngb-pagination>
    </div>
  </div>
  <br>
  <div>
    <h5>Contratos</h5>
    <div class="card" *ngFor="let contrato of licitacao?.contratosLicitacao ; index as i;">
```
Colocar o operador ? também evitou alguns erros aqui. Aí tava acontecendo também?

Reply:
Não tava acontecendo não, mas coloquei agora pra evitar